tiingo/api.py

```diff
@@ -178,7 +178,7 @@ def get_ticker_price(self, ticker,
                 fmt (string): 'csv' or 'json'
                 frequency (string): Resample frequency
         """
-        url = self._get_url(ticker, frequency)
+        url = self._get_price_url(ticker, frequency)
```

Comment:
Was curious about the motivation for renaming this method, since it doesn't look like the new crypto functionality is related to it. Would it be OK to leave this renaming for a future PR?

Reply:
At the beginning, I was trying to create a function like  `_get_crypto_url` so I thought I change this function's name to be more specific. But at the end, I did not end up writing that function, so I will revert this change in the next commit.